frontend/src/pages/NewTeacher.vue

```diff
@@ -1,352 +1,350 @@
-          <!--.prevent prevents the default submit behaviour and executes createTeacher instead -->
+          <form
+            method="POST"
+            action="/api/students"
+            @submit.prevent="createTeacher"
+          >
+            <!--.prevent prevents the default submit behaviour and executes createTeacher instead -->
             <section>
-                <b-field label="Name" class="half-width">
-                    <b-input v-model="name" name="Name"></b-input>
+              <b-field label="Name" class="half-width">
+                <b-input v-model="name" name="Name"></b-input>
+              </b-field>
+
+              <b-field label="Phone Number" class="half-width">
+                <b-input v-model="PhoneNumber" type="string" value="">
+                </b-input>
+              </b-field>
+
+              <b-field label="Email Address" class="half-width">
+                <b-input v-model="Email" type="string" value=""> </b-input>
+              </b-field>
+
+              <b-field label="Source" class="half-width">
+                <b-input v-model="source" placeholder="Optional"> </b-input>
+              </b-field>
+              <b-field grouped>
+                <b-field label="Native Language" class="half-width">
+                  <b-autocomplete
+                    v-model="nativeLanguage"
+                    field="NativeLanguage"
+                    ref="languageComplete"
+                    :data="languages"
+                    placeholder="e.g. Bengali"
+                    @typing="filteredLanguageDataArray"
+                    @select="
+                      (option) =>
+                        (selected.NativeLanguage = option.NativeLanguage)
+                    "
+                  >
+                    <template slot="header">
+                      <a @click="showAddLanguage">
+                        <span> Add new... </span>
+                      </a>
+                    </template>
+                  </b-autocomplete>
                 </b-field>
 
-                <b-field label="Phone Number" class="half-width">
-                    <b-input v-model="PhoneNumber" type="string"
-                        value="">
-                    </b-input>
+                <b-field label="English Proficiency" class="half-width">
+                  <b-select
+                    v-model="EnglishProficiency"
+                    placeholder="Select one"
+                    expanded
+                  >
+                    <option value="No">No (Unable to understand at all)</option>
+                    <option value="Little">
+                      Little (Able to understand simple words)
+                    </option>
+                    <option value="Simple">
+                      Simple (Able to speak full sentences)
+                    </option>
+                    <option value="Intermediate">
+                      Intermediate (Able to understand simple words)
```

Comment:
TODO: update these options as specified in https://github.com/TechLadies/irrc-madwish/issues/85

Reply:
Noted, will update as part of a separate PR as discussed